src/util/azure_utils.ts

```diff
@@ -45,20 +55,19 @@ export async function getEthereumAddress(publicKey: Buffer): Promise<string> {
  */
 export async function sign(digest: Buffer,
     keyVaultCredentials: AzureKeyVaultCredentials): Promise<SignResult> {
-  const credentials = new ClientSecretCredential(
-      keyVaultCredentials.tenantId, keyVaultCredentials.clientId,
-      keyVaultCredentials.clientSecret);
-
   try {
-    const keyVaultUrl = `https://${keyVaultCredentials.vaultName}.vault.azure.net`;
-    const client = new KeyClient(keyVaultUrl, credentials);
+    const credentials = new ClientSecretCredential(
+        keyVaultCredentials.tenantId,
+        keyVaultCredentials.clientId,
+        keyVaultCredentials.clientSecret,
+    );
+    const client = new KeyClient(keyVaultCredentials.vaultUrl, credentials);
     const keyObject = await client.getKey(keyVaultCredentials.keyName);
     const cryptographyClient = new CryptographyClient(keyObject, credentials);
```

Comment:
I guess this is coming from AzureKeyVault. can we change the instance name from `cryptographyClient` to something else?

Reply:
akvCryptoClient?

Reply:
renamed to keyVaultCryptoClient